parsec/core/gui/workspaces_widget.py

```diff
@@ -248,7 +286,7 @@ def goto_file_clicked(self):
                 ts = button.workspace_fs.decrypt_timestamp(addr)
 
                 # Mount workspace at the link's creation time
-                if not self.core.mountpoint_manager.is_workspace_mounted(button.workspace_fs, ts):
+                if not self.core.mountpoint_manager.is_workspace_mounted(button.workspace_fs, ts):  # type: ignore[arg-type]
```

Comment:
The ignore type is probably not needed, it looks more like a bug here, i guess it suppose to be:
`button.workdpace_fs.workspace_id`